client/admin/src/views/Orders.vue

```diff
@@ -0,0 +1,250 @@
+    this.loadOrders();
+  }
+
+  public filterOrders() {
+    /**
+     * I have an original array (this.orders) and another array (this.ordersDisplay) , the orders array will never be
+     * changed, only the selected values from the settings panel will be used to filter data from the original orders array
+     * back into the ordersDisplay array and then appear in the ui to the user.
+     *
+     * The orders array will keep all the data from graphql query that returns the order attributes. The ordersDisplay will change
+     * each time the user selects more columns or removes columns from the setttings tab. Each column value is a property in the poduct
+     * object. So for each order object in the orders array we filter out only the desired keys from the objects and clone them
+     * into the ordersDisplay array.
+     */
+    let tableColums = this.tableColums;
+
+    this.ordersDisplay = this.orders.map(function (order: any) {
+      return Object.keys(order)
+        .filter((k) => tableColums.includes(k))
+        .map((k) => Object.assign({}, { [k]: transform(order[k]) }))
+        .reduce((res, o) => Object.assign(res, o), {});
+    });
+  }
+
+  async loadOrders() {
+    this.ordersResults({
+      first: this.paginatorInfo.first,
+      page: this.paginatorInfo.currentPage,
+    })
+      .then(
+        (data: {
+          data: { orders: { data: object[]; paginatorInfo: Paginator } };
+          errors: [{ debugMessage: string }];
+        }) => {
+          console.log(data);
+
+          try {
+            let error = data.errors[0].debugMessage;
+            this.errors.push(error);
+            console.log(error);
+            return;
+          } catch (error) {
+            // no error
+          }
+
+          let dataStore = this.cleanupData(data.data.orders.data);
+          this.orders = dataStore;
+          this.ordersAttributes = Object.keys(dataStore[0]);
+
+          // Assign paginator information, and set reload to false to prevent infinite loop
+          // as this component also watches for state changes in the paginator vuex state
+          // because the table component changes the paginator state and this component fetches the result of that change
+          let paginatorUpdate: {} = data.data.orders.paginatorInfo;
+          this.updatePaginatorInfo({
+            ...{ reload: false },
+            ...paginatorUpdate,
+          });
+
+          // This is to have a few columns displaying on initial view, by pushing the column keys into
+          // tableColumns
+          if (this.tableColums.length === 0) {
+            this.ordersAttributes.slice(0, 4).forEach((key: string) => {
+              this.tableColums.push(key);
+            });
+          }
+
+          // Populate column data
+          this.filterOrders();
+
+          return Promise.resolve();
+        }
+      )
+      .catch((error: string) => {
+        console.log(error);
+      });
+  }
+
+  cleanupData(data: any) {
```

Comment:
@AlexeiaW I think I saw this method in your other PR. I notice in this answer there's possibly a way to omit this from Apollo altogether, but not sure what you think. Is it useful to have this data available for debugging?

https://stackoverflow.com/a/51380645/558287

If you look at the top method:

```
apollo.create({
  link: http,
  cache: new InMemoryCache({
    addTypename: false
  })
});
```

Otherwise I'd suggest we just move this into a helper method.

Reply:
Makes sense to move it to a helper for sure